packages/tests/server/initTRPC.test.ts

```diff
@@ -39,6 +46,42 @@ test('custom transformer', () => {
   ).not.toMatchTypeOf<DefaultDataTransformer>();
 });
 
+test('default namespace delimiter', () => {
+  const t = initTRPC
+    .context<{
+      foo: 'bar';
+    }>()
+    .create();
+  const router = t.router({});
+
+  expectTypeOf<typeof router._def._config.$types.ctx>().toMatchTypeOf<{
+    foo: 'bar';
+  }>();
+
+  expectTypeOf(
+    t._config.namespaceDelimiter,
+  ).toMatchTypeOf<DefaultNamespaceDelimiter>();
+  expectTypeOf(
+    router._def._config.namespaceDelimiter,
+  ).toMatchTypeOf<DefaultNamespaceDelimiter>();
+});
+
+test('custom namespace delimiter', () => {
```

Comment:
we don't need this test here since it's the same as the one in `namespaceDelimiter.test.ts`

Reply:
Thank you! 🙏 

Agree, redundant after the suite was added.

Removed in latest commit.